.github/workflows/main.yaml

```
@@ -0,0 +1,44 @@
+---
+name: Continuous integration
+
+on:
+  push:
+  pull_request:
+
+env:
+  HAS_SECRETS: ${{ secrets.HAS_SECRETS }}
+
+jobs:
+  main:
+    runs-on: ubuntu-20.04
+    name: Continuous integration
+    timeout-minutes: 10
+#    if: "!startsWith(github.event.head_commit.message, '[skip ci] ')"
+
+    steps:
+      # Versions
+      - run: docker --version
+      - uses: actions/checkout@v2
+
+      - uses: camptocamp/initialise-gopass-summon-action@v2
+        with:
+          ci-gpg-private-key: ${{secrets.CI_GPG_PRIVATE_KEY}}
+          github-gopass-ci-token: ${{secrets.GOPASS_CI_GITHUB_TOKEN}}
+          patterns: docker
+        if: env.HAS_SECRETS == 'HAS_SECRETS'
+
+      - run: echo "${HOME}/.local/bin" >> ${GITHUB_PATH}
+      - run: python3 -m pip install --user --requirement=ci/requirements.txt
+
+      - name: Checks
+        run: c2cciutils-checks
+
+      - run: make pull
+      - run: make build
+      - run: make acceptance
+      - run: (cd acceptance_tests/ && docker-compose logs)
```

you can rewrite like this:

```suggestion
      - name: acceptance tests logs
        run: docker-compose logs
        working-directory: acceptance_tests
```